trace/trace.go

```diff
@@ -111,9 +118,11 @@ type StartOptions struct {
 	// when there is a non-remote parent, no new sampling decision will be made:
 	// we will preserve the sampling of the parent.
 	Sampler Sampler
-}
 
-// TODO(jbd): Remove start options.
+	// SpanKind represents the kind of a span. If none is set,
+	// SpanKindUnspecified is used.
```

Comment:
I would specify that these must be one of the SpanKindUnspecified, SpanKindServer, SpanKindClient

Reply:
Done